jobbergateapi2/tests/test_pagination.py

```diff
@@ -19,17 +19,15 @@ def test_pagination_with_default_values():
     """
     pagination = Pagination()
 
-    assert pagination.q is None
     assert pagination.limit == 10
-    assert pagination.offset == 0
+    assert pagination.skip == 0
 
 
 def test_pagination_with_custom_values():
     """
     Test the pagination with custom values
     """
-    pagination = Pagination(q="value", limit=100, offset=1)
+    pagination = Pagination(limit=100, skip=1)
 
-    assert pagination.q == "value"
     assert pagination.limit == 100
```

Comment:
This tests that the limit and the skip are set in the Pagination object, but does not check the functionality. It would be worth checking if the limit actually limits values returned, and that the offset correctly selects the first record to return.

Reply:
This test is exactly for that: https://github.com/omnivector-solutions/jobbergate-api-fastapi/blob/a984be82d72ac6d3ad3de876ead9c8aa2c49b251/jobbergateapi2/tests/apps/users/test_routers.py#L38